spec/models/comment_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
```

Comment:
- [ ] I could be to run perfectly the tests because An error has occurred said: Table 'comments' has no foreign key for users. Kindly fix this 👍🏻 

![Screenshot from 2022-11-25 08-38-03](https://user-images.githubusercontent.com/79058364/203917224-93099a93-3068-401d-9cb2-93b35cddf732.png)